### DISCUSSION TITLE
Problem to insert librarie

### DISCUSSION THREAD

**Discussion by ronceray35** [cutelyst/simple-mail#96] [General]
Hello all

how insered the library in my project
Tks

**Comment by dantti** [comment]
Use find_package() or fetch_content()
https://github.com/cutelyst/cutelyst/blob/master/Cutelyst/Plugins/View/CMakeLists.txt#L12

**Comment by ronceray35** [comment]
i m very sorry but i must download the files and after. i don 't understand how insert the library with my project.

thx
